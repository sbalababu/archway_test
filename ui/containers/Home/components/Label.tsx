import * as React from 'react';

interface Props {
  style?: React.CSSProperties;
  children: any;
}

const Label = ({ style, children }: Props) => (
  <div
    style={{
      fontSize: 16,
      textTransform: 'uppercase',
      letterSpacing: 1,
      fontWeight: 200,
      display: 'flex',
      alignItems: 'center',
      justifyContent: 'center',
      ...style,
    }}
  >
    {children}
  </div>
);

export default Label;
