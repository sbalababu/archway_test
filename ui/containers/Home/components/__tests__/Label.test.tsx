import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Label from '../Label';

describe('Label', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Label children="Label" />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
