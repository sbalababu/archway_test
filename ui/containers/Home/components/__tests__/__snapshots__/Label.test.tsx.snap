// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "fontSize": 16,
      "fontWeight": 200,
      "justifyContent": "center",
      "letterSpacing": 1,
      "textTransform": "uppercase",
    }
  }
>
  Label
</div>
`;
